src/py-opentimelineio/opentimelineio/media_linker.py

```diff
@@ -148,6 +149,15 @@ def link_media_reference(self, in_clip, media_linker_argument_map=None):
             media_linker_argument_map=media_linker_argument_map
         )
 
+    def plugin_info_map(self):
+        """Adds extra adapter-specific information to call to the parent fn."""
+
+        result = super(MediaLinker, self).plugin_info_map()
+        mod_doc = [result['doc'], ""]
```

Comment:
Wondering about error handling and how the super plugin_info_map() can sometimes return a different dict with only an "ERROR" key on an ImportError.

Reply:
yup!  also a good note.